### PR TITLE
Updated to the latest Rust

### PR DESCRIPTION
**Pull request by Divius**
Seems to be a hot time, let's update again :D

### CODE REVIEW

File: src/bencode.rs [hunk 1]
```diff
@@ -1944,7 +1945,7 @@ mod bench {
 
     #[bench]
     fn encode_large_vec_of_uint(bh: &mut Bencher) {
-        let v = Vec::from_fn(100, |n| n);
+        let v: Vec<u32> = iter::range(0, 100).map(|n| n).collect();
```
Comment (arjantop) [inline src/bencode.rs]: Map is not needed here.

File: src/bencode.rs [hunk 2]
```diff
@@ -1959,7 +1960,7 @@ mod bench {
 
     #[bench]
     fn decode_large_vec_of_uint(bh: &mut Bencher) {
-        let v = Vec::from_fn(100, |n| n);
+        let v: Vec<u32> = iter::range(0, 100).map(|n| n).collect();
```
Comment (arjantop) [inline src/bencode.rs]: Map is not needed here.

File: src/bencode.rs [hunk 3]
```diff
@@ -1936,6 +1936,7 @@ mod bench {
     use self::test::Bencher;
 
     use std::io;
+    use std::iter;
```
Comment (arjantop) [inline src/bencode.rs]: `range` is already in the prelude, `use` is not necessary. Use `range` directly.

**Comment by Divius** [comment]
Right, sorry. Fixed.

**Pull request merged**